app/views/local_authorities/index.html.erb

```diff
@@ -10,7 +10,9 @@
   <table class="table table-striped table-hover table-bordered contacts-table" data-module="filterable-table">
     <thead>
       <tr class="table-header">
-        <th>Local Authorities (<%= @authorities.count %>)</th>
+        <th>
+          <div class='count'><span><%= @authorities.count %></span> local authorities</div>
```

Comment:
^ thanks, fixed :) 

So there's a spec which tests the wording that this commit changes. It gets updated in [another commit in this PR](https://github.com/alphagov/local-links-manager/pull/71/commits/c40c642b7615bee8347b3e93521bc662dd37536b). Might be worth either pulling that test change into this commit, or rolling this commit into the other one...